### ISSUE TITLE
Do we need a smaller threshold for diagonal movement?

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#331]
From #322, I am wondering if the experience would benefit from a lower threshold to trigger Diagonal movement descriptions on releases.

They diagonal descriptions are working great for releases on Sweater!

Just wondering if we more detail for releases where the balloon travels across the Play Area.

**Comment by emily-phet** [comment]
@terracoda can we talk about this tomorrow?

**Comment by terracoda** [comment]
@emily-phet, yes, that is not a problem. I consider this issue a design tweak, and not a huge prioroty.

Until all the alerts (specifically, balloon movement alerts) are clearly labeled and implemented, it is difficult to say exactly where improvements would be most beneficial.

**Comment by terracoda** [comment]
@emily-phet, I have converted this issue into a project card and will re-open if needed after further testing.

Closing this issue for now.
